src/site/pages/guidelines.md

```diff
@@ -63,74 +63,92 @@ PatternFly follows a two-layer theming system where **global variables** always
 
-- They follow this formula `$pf-c-block__element--modifier--state--PropertyCamelCase`.
-  - The `pf-c-block__element--modifier` is the selector name is something like `pf-c-alert__actions`;
-  - a `state` is something like `hover` or `active`;
-- The variable always has a default value.
-- The value of component scoped variables is **always** defined by a global variable.
+- They follow this general formula `--pf-c-block__element--modifier--state--breakpoint--pseudo-element--PropertyCamelCase`.
+  - `pf-c-block` refers to the block, usually the component or layout name (i.e., `pf-c-alert`).
+  - `__element` refers to the element inside of the block (i.e., `__title`).
+  - `modifier` refers to a modifier class such as `.pf-m-danger`, and is prefixed with `m-` in the component variable (i.e., `m-danger`)
+  - `state` is something like `hover` or `active`.
+  - `breakpoint` is a media query breakpoint such as `sm` for `$pf-global--breakpoint--xs`.
+  - `pseudo-element` is one of either `before` or `after`.
+- The value of a component variable is **always** defined by a global variable.
+- It's possible to include multiple elements, modifiers, states, and breakpoints in a single component variable.
+- The order of elements, modifiers, sttates, and breakpoints in the variable name should match the selector order.
 
 For example:
 
 ```scss
 /* Component scoped variables are always defined by global variables */
-  --pf-alert--Padding: var(--pf-global--spacer--xl);
-  --pf-alert--hover--BackgroundColor: var(--pf-global--BackgroundColor--hover);
-  --pf-alert__title--FontSize: var(--pf-global--FontSize--2xl);
+  --pf-c-alert--Padding: var(--pf-global--spacer--xl);
+  --pf-c-alert--hover--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-alert__title--FontSize: var(--pf-global--FontSize--2xl);
 
 /* --block--PropertyCamelCase */
-.alert {
-  padding: var(--pf-alert--Padding);
+.pf-c-alert {
+  padding: var(--pf-c-alert--Padding);
 }
 
 /* --block--state--PropertyCamelCase */
-.alert {
+.pf-c-alert {
   &:hover {
-    background-color: var(--pf-alert--hover--BackgroundColor);
+    background-color: var(--pf-c-alert--hover--BackgroundColor);
   }
 }
 
 /* --block__element--PropertyCamelCase */
-.alert__title {
-  font-size: var(--pf-alert__title--FontSize);
+.pf-c-alert__title {
+  font-size: var(--pf-c-alert__title--FontSize);
 }
-```
 
-This setup allows for consistency across components, it generates a common language between designers and developers, and it gives granular control to authors.
+/* A more complex example */
+.pf-c-switch {
+  @media (max-width: $pf-global--breakpoint--sm) {
+    .pf-c-switch__input {
+      &:disabled {
+        ~ .pf-c-switch__toggle {
+          &::before {
+            background-color: var(--pf-c-switch--sm__input--disabled__toggle--before--BackgroundColor);
```

Comment:
should this be `--pf-c-switch__input--disabled__toggle--sm--before--BackgroundColor`?

Reply:
I don't think so? because the `sm` breakpoint is on `.pf-c-switch`, not `.pf-c-switch__toggle`. As in you could have other stuff like color or font size on `.pf-c-switch` at that breakpoint with vars like `--pf-c-switch--sm--Color, --pf-c-switch--sm-FontSize`. Seems like the breakpoint should remain static on the thing it's assigned to, but I'm open to other suggestions.

Reply:
ah right, ok that works for me.